docs/dev/integrate_operator_how_to.md

```diff
@@ -0,0 +1,53 @@
+## How to add a new operator to the installer?
+
+### Goals:
+To follow this document and be able to
+ - Get an operator’s configuration files be created for installation
+ - Get an operator’s manifest files be put in place for cluster to pick up
+ - Get the config files and manifest files be created as resources in the new cluster, in the appropriate order
+
+### Details
+The source code for the installer is here: https://github.com/openshift/installer. We will refer to this repo as installer-src for this document.
+
+One can can classify all the manifest/config files of an operator in two categories:
+ - Static
+ - Templatized
+ 
+The static ones are clearly just byte blobs and the templates are ones that have some variable that needs to be filled up. Typically the variables that need filling up will come from user preferences as specified in the install config (e.g. cluster name, cluster domain, service cidr etc). Other dependencies could be TLS cert keys for example.
+
+#### The recommended way
+Take the static files and ready it for the Cluster Version Operator (CVO) to pick it up as its payload. See this document:
+https://docs.google.com/document/d/1CGZVEyuloZ9oD4NUArW6dnEpi0WFc6BP2tqPSwFZTCY/edit#heading=h.1zgrwxmpgxbr
+
+Upshot of the above document is that the operator should keep its static manifest files in manifests/ folder of the operator repository and manage image references through an image-references file. The order of creation of the files is alphabetical, so the files should be numbered like below to effectively create the service account before the deployment.
```

Comment:
> Upshot of the above document is that the operator should keep its static manifest files in manifests/ folder of the operator repository and manage image references through an image-references file.

The document above requires more than keeping in /manifests of the repo. I would recommend not duplicate here.

> The order of creation of the files is alphabetical, so the files should be numbered like below to effectively create the service account before the deployment.

`ClusterVersionOperator` applies the manifests to cluster in alphabetical order. So it is recommended that manifests be order of dependencies.
For example, 
* a deployment requires service account, roles etc and,
* everything requires the namespace to exist.

So the order of the manifests should be enforces like:

Reply:
> The document above requires more than keeping in `/manifests` of the repo. I would recommend not duplicate here.

Agreed on not duplicating here.  And any references to this that we keep should talk about the operator *images*; the operator Git repository doesn't come into it.

But if we punt this portion of the docs to external docs (whether in a Google Doc or github.com/openshift/cluster-version-operator Markdown), what's left to talk about here?  Just:

> Almost everyone should use [external docs](link-to-wherever-they-are).  {details about when that won't work}.  {where to put things when you add them to this repo}.